package.json

```diff
@@ -14,15 +14,16 @@
   "main": "index.js",
   "scripts": {
     "test": "lab -cv",
-    "lint": "jshint test index.js --reporter node_modules/jshint-stylish/stylish.js --exclude node_modules"
+    "lint": "jshint test index.js --reporter node_modules/jshint-stylish/index.js --exclude node_modules"
```

Comment:
@erikkemperman a quick heads-up on these.  I am looking to switch the gulp-related repos to the gulpjs organization but I was waiting until I got your PRs merged.  Should I continue to wait or just start transferring them whenever?